src/components/authorization/model/resource-map.ts

```diff
@@ -1,100 +1,6 @@
-import { Budget, BudgetRecord } from '../../budget/dto';
-import { Ceremony } from '../../ceremony/dto';
-import { Changeset } from '../../changeset/dto';
-import { Comment, Commentable, CommentThread } from '../../comments/dto';
-import {
-  IEngagement as Engagement,
-  InternshipEngagement,
-  LanguageEngagement,
-} from '../../engagement/dto';
-import { EthnoArt } from '../../ethno-art/dto';
-import { FieldRegion } from '../../field-region/dto';
-import { FieldZone } from '../../field-zone/dto';
-import {
-  Directory,
-  File,
-  IFileNode as FileNode,
-  FileVersion,
-} from '../../file/dto';
-import { Film } from '../../film/dto';
-import { FundingAccount } from '../../funding-account/dto';
-import { EthnologueLanguage, Language } from '../../language/dto';
-import { Location } from '../../location/dto';
-import { Organization } from '../../organization/dto';
-import { Partner } from '../../partner/dto';
-import { Partnership } from '../../partnership/dto';
-import {
-  FinancialReport,
-  NarrativeReport,
-  IPeriodicReport as PeriodicReport,
-} from '../../periodic-report/dto';
-import { Post } from '../../post/dto';
-import {
-  DerivativeScriptureProduct,
-  DirectScriptureProduct,
-  OtherProduct,
-  Producible,
-  Product,
-} from '../../product/dto';
-import { ProjectChangeRequest } from '../../project-change-request/dto';
-import {
-  InternshipProject,
-  IProject as Project,
-  TranslationProject,
-} from '../../project/dto';
-import { ProjectMember } from '../../project/project-member/dto';
-import { Story } from '../../story/dto';
-import { User } from '../../user/dto';
-import { Education } from '../../user/education/dto';
-import { Unavailability } from '../../user/unavailability/dto';
 import { AssignableRoles } from '../dto/assignable-roles';
-import { BetaFeatures } from '../dto/beta-features';
 
 /** @deprecated Use {@link import('~/core').ResourcesHost.getMap} instead */
 export const LegacyResourceMap = {
-  Commentable,
-  Budget,
-  BudgetRecord,
-  Ceremony,
-  Changeset,
-  Comment,
-  CommentThread,
-  Directory,
-  Education,
-  Engagement,
-  EthnoArt,
-  EthnologueLanguage,
-  FieldRegion,
-  FieldZone,
-  File,
-  FileNode,
-  FileVersion,
-  Film,
-  FundingAccount,
-  InternshipEngagement,
-  Language,
-  LanguageEngagement,
-  Location,
-  Organization,
-  Partner,
-  Partnership,
-  Post,
-  Producible,
-  Product,
-  DirectScriptureProduct,
-  DerivativeScriptureProduct,
-  OtherProduct,
-  Project,
-  TranslationProject,
-  InternshipProject,
-  ProjectMember,
-  PeriodicReport,
-  FinancialReport,
-  NarrativeReport,
-  ProjectChangeRequest,
-  Story,
-  Unavailability,
-  User,
   AssignableRoles,
```

Comment:
@CarsonF - I am confused a little bit by this AssignableRoles. It does not fit the Resource shape (however, some of its properties fit the resource shape) - So if I try to move this to the decorator approach, it blows up a lot of stuff. But I am curious why this is actually a part of the Legacy Map in the first place with how different it is than the rest of the items I have removed.

Reply:
What blows up?

Reply:
it doesn't recognize that the decorator is a function when removing this specific resource. 

![image](https://github.com/SeedCompany/cord-api-v3/assets/37881163/6ffb8fc7-50fc-4767-9858-520da010cd62)

Reply:
Trying to trace the threads of how this relates to all the other things that fail (because for science, I went and undid some of the changes for these resources that start throwing these errors) and essentially none of them will work once this one is removed.

Reply:
Oh this is circular dependency problem with imports. These are long standing rough edges.
Try importing from `~/core/resources` so `core` as a whole is not referenced.

Also if you rebase for ESM that's now in develop, the ESM loader calls out explicitly where the import is a problem.